test/services/user/services/AdminUsers.test.js

```diff
@@ -457,4 +491,38 @@ describe('AdminTeachersService', () => {
 		expect(idsOk).to.include(teacherWithConsent._id.toString());
 		expect(idsOk).to.not.include(teacherWithoutConsent._id.toString());
 	});
+
+	it('does not allow user creation if email already exists', async () => {
+		const admin = await testObjects.createTestUser({ roles: ['administrator'] });
+		const params = await testObjects.generateRequestParamsFromUser(admin);
+		const mockData = {
+			firstName: 'testFirst',
+			lastName: 'testLast',
+			email: 'test@de.de',
+			roles: ['teacher'],
+			schoolId: admin.schoolId,
+		};
+		// creates first teacher with unique data
+		await adminTeachersService.create(mockData, params);
+		// creates second teacher with existent data
+		await adminTeachersService.create(mockData, params).catch((err) => {
+			expect(err.code).to.equal(400);
+		});
+	});
+
+	it('does not allow user creation if email is disposable', async () => {
+		const admin = await testObjects.createTestUser({ roles: ['administrator'] });
+		const params = await testObjects.generateRequestParamsFromUser(admin);
+		const mockData = {
+			firstName: 'testFirst',
+			lastName: 'testLast',
+			email: 'disposable@20minutemail.com',
+			roles: ['teacher'],
+			schoolId: admin.schoolId,
+		};
+		// creates teacher with disposable email
+		await adminTeachersService.create(mockData, params).catch((err) => {
+			expect(err.message).to.equal('EMAIL_DOMAIN_BLOCKED');
```

Comment:
I think the test will be green if the service doesn't throw, so that's not good. I think you should do something like [this](https://github.com/schul-cloud/schulcloud-server/blob/8f13b0a142382977d41987c986e641cc2df52b84/test/services/news/index.test.js#L46)